ui/app/components/modals/new-account-modal.js

```diff
@@ -1,17 +1,105 @@
-const Component = require('react').Component
+const { Component } = require('react')
+const PropTypes = require('prop-types')
 const h = require('react-hyperscript')
-const inherits = require('util').inherits
-const connect = require('react-redux').connect
+const { connect } = require('react-redux')
 const actions = require('../../actions')
+const R = require('ramda')
+
+const getNewAccountName = accountNames => {
+  let accountNumber = 1
+
+  while (accountNames.indexOf(`Account ${accountNumber}`) > -1) {
```

Comment:
I think this would have unexpected UX in this flow:
1. Create new account (Account 1)
2. Rename Account 1 to `asdfasd`
3. Create another account with no name
4. The new account is Account 1 again

Perhaps we should just use `accountNames.length + 2`?

Reply:
We should also cross reference how default name are generated in the background process

Reply:
That was actually the flow I was going for, since I saw "Account 1-X" as placeholder names so they'd be in sequential order. I can use `accountNames.length + 1` instead though.